selection_fields.py

```diff
@@ -0,0 +1,176 @@
+import sublime
+import sublime_plugin
+
+_ST3 = sublime.version() >= "3000"
+
+# highlight pushed region options
+if _ST3:
+    _FLAGS = sublime.DRAW_EMPTY | sublime.DRAW_NO_FILL
+else:
+    _FLAGS = sublime.DRAW_EMPTY | sublime.DRAW_OUTLINED
+
+_SCOPE = "comment"
+
+
+def _change_selection(view, regions, pos):
+    """Extract the next selection, push all other fields."""
+    sel = regions[pos]
+    del regions[pos]
+    # push the fields to the view, kwargs for ST3 and pos args for ST2
+    if _ST3:
+        view.add_regions("meu_sf_stored_selections", regions, scope=_SCOPE,
+                         flags=_FLAGS)
+    else:
+        view.add_regions("meu_sf_stored_selections", regions, _SCOPE, _FLAGS)
+    view.settings().set("meu_sf_current_cursor", pos)
+    sel_regions = [sel]
+    # add a feedback to the statusbar
+    view.set_status("meu_field_message",
+                    "Sel-Field {0} of {1}".format(pos + 1, len(regions) + 1))
```

Comment:
I would prefer to avoid abbreviations here. How about `Selection-Field {0} of {1}`?